tox.ini

```diff
@@ -2,5 +2,5 @@
 max-line-length=120
 application_import_names=bot
 exclude=.venv
-ignore=B311,W503,E226
+ignore=B311,W503,E226,S311
```

Comment:
What error is this? Why are we ignoring it?

Reply:
it stops travis from whining about `random.choice` being insecure

Reply:
Ah, okay